tests/classify_test.py

```diff
@@ -130,6 +130,22 @@ def test_application_directories(in_tmpdir, no_empty_path):
     assert ret is ImportType.APPLICATION
 
 
+@pytest.mark.xfail(
+    sys.platform.startswith('linux'),
```

Comment:
since python3 all linux platforms report `sys.platform` as `'linux'` -- the startswith hack is no longer needed since python2 is not supported

Reply:
Merged with the other xfail marker: https://github.com/asottile/aspy.refactor_imports/pull/53/commits/3685fe2020e9a31473301bd0aaf111fa53a7b0a7